modules/Vtiger/inventoryfields/Subunit.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Inventory Subunit Field Class.
+ *
+ * @package   InventoryField
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Arkadiusz Adach <a.adach@yetiforce.com>
```

Comment:
This can be changed to your data :)

Reply:
oh thx :)
I'll change only author details: I subscribe YetiForce Public License 3.0

Reply:
done